build.gradle

```diff
@@ -24,5 +24,5 @@ dokka {
     moduleName = 'outcobra-server'
     outputFormat = 'javadoc'
     outputDirectory = "$buildDir/javadoc"
-    sourceDirs = files('backend/src/main/kotlin/outcobra', 'backend-model/src/main/java/outcobra')
+    sourceDirs = files('backend/src/main/kotlin/outcobra', 'backend-outcobra.server.model/src/main/java/outcobra')
```

Comment:
Intended?

Reply:
No I used the global replace function. I'll fix it.